data/update.py

```diff
@@ -46,7 +46,7 @@
 # domain-scan information
 SCAN_TARGET = os.path.join(this_dir, "./output/scan")
 SCAN_COMMAND = os.environ.get("DOMAIN_SCAN_PATH", None)
-SCANNERS = os.environ.get("SCANNERS", "inspect,analytics,sslyze,tls")
+SCANNERS = os.environ.get("SCANNERS", "pshtt,analytics,sslyze,inspect,tls")
```

Comment:
I'm leaving the `inspect` scanner in the production environment and scan process, so that we still have `inspect.csv` and `cache/inspect/` files saved and uploaded to S3 for a while, to compare as necessary. If there are no problems after a month or two, we can drop `inspect` from the scan workflow. It shouldn't add much time to the overall scan process.